standard-hackathon-rules.md

```diff
@@ -29,14 +29,14 @@ Remember that hackathons are like marathons. Some people go to compete but most
 
 After hacking finishes, teams will show their projects each other and to the judges.
 
-You are strongly encouraged to present a demo of what you have built. Pitches or presentations are discouraged. You are not judged on the quality of your pitch or the quality of your idea. As you are judged on what you built, you'll only hurt yourself by not showing a demo.
+You are encouraged to present a demo what you have done even if your hack is broken or you weren’t able to finish. Pitches or presentations without a technical demo are discouraged. You are not judged on the quality of your pitch or the quality of your idea. As you are judged on what you built, you'll only hurt yourself by not showing a demo.
```

Comment:
Grammatically better can be:

`You are encouraged to present a demo of what you have done`

**of** should be added here

Reply:
i agree!